docker-tools/Dockerfile-from-conda

```diff
@@ -2,16 +2,16 @@
 FROM frolvlad/alpine-miniconda3
 LABEL org.pydata.bokeh.maintainer="Karel-van-de-Plassche <karelvandeplassche@gmail.com>"
 
+RUN apk add --no-cache \
+    bash
```

Comment:
why the line continuation?

Reply:
It should be `RUN apk add --no-cache bash` so I need a line continuation, correct? I got this from "Sort multi-line arguments" from https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

Reply:
I see why they do that when there is a list I just question whether we will ever have a list. I am fine with it either way

Reply:
Ah yeah, we will probably not have a list per se here, but like this I could easily copy from other scripts/templates ;) I'll keep it for this reason.